0-bootstrap/README-Jenkins.md

```diff
@@ -284,6 +284,12 @@ Here you will configure a VPN Network tunnel to enable connectivity between the
 
 1. You can now move to the instructions in the step [1-org](../1-org/README.md).
 
+### Optional Variables
+
+Some variables used to deploy the step have default values. Check if you want to change those values before doing the deploy:
+
+- **skip_gcloud_download:** By default this option is `true`. If you change to false it will download gcloud. The default value assumes gcloud is already available outside the module.
```

Comment:
It is usually easier to follow positive statements like:
`download_gcloud = true` -> then it means `gcloud` will be downloaded.

So, is `skip_gcloud_download`is only used here and it is not a variable that we use in other modules, I would change `skip_gcloud_download` to `download_gcloud`

Reply:
I agree, but I think would be better to fix this in another PR. Thanks!

Reply:
`skip_gcloud_download` is from `gcloud` module which exposes `skip_download`. I think we kept it because default behaviour of gcloud module was to enable users to download and use gcloud. With the 2.0 release we default `skip_download` to true and expose more controls, so we wont need to plumb this. But yes, I agree with @atos-cit we can take care of that bump and minor refactor in a separate PR.